### PR TITLE
[npm] Updating angular packages to the latest stable version

### PR DESCRIPTION
**Pull request by brianjjones**
Signed-off-by: Brian J Jones <brian.j.jones@intel.com>

### CODE REVIEW

**Comment by brianjjones** [comment]
@nagineni Could you take a look at this PR and tell me if it looks good to you?  Thanks.

**Comment by nagineni** [comment]
@brianjjones deps in package.json seems to be good, but I'm seeing some diff in the package-lock.json after I run 'npm install' in my env. I have node LTS v8.9.3 in my env.

This is what I did:
1. Applied this PR
2. cp package-lock.json to package-lock.json.orig
3. removed node_modules and cleared the npm cache
4. Run 'npm install'

Compared newly created package-lock.json with package-lock.json.orig and found some changes.

**Comment by brianjjones** [comment]
@nagineni I think that's because currently we don't have an 'engines' field in our package.json file that specifies the node version.  I can certainly add it, probably wouldn't be a bad idea.  Do you know a good version to require?  Sorry I'm still pretty new to npm :)

**Comment by nagineni** [comment]
@brianjjones Like it mentioned in the README - ... to take advantage of the complete functionality we strongly recommend node >=v6.5.0 and npm >=3.10.3. So, you can add both in the engines field in package.json.

**Comment by brianjjones** [comment]
Updated with an engine field for a node and npm

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "zephyrjs-ide",
-  "version": "0.21.0-alpha",
```
Comment (nagineni) [inline package.json]: Version "0.22.0-alpha" is already in the master. We update the version when we are about to make a new release.

File: package.json [hunk 2]
```diff
@@ -48,6 +48,10 @@
   },
   "author": "Salvatore Iovene <salvatore.iovene@intel.com>",
   "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.5.0",
+     "npm": ">=3.10.3"
```
Comment (nagineni) [inline package.json]: minor nit: Fix the indentation.

**Comment by nagineni** [comment]
LGTM

I've not tested this with the latest commit. Before merging the PR, please make sure no local diff after running 'npm install'.

**Comment by xiaoyu3x** [comment]
@brianjjones 
I just verified this PR and 'npm install' executed successfully.

**Pull request merged**